builds/any/rootfs/buster/standard/standard.yml

```diff
@@ -24,21 +24,21 @@ Multistrap:
     noauth: true
     explicitsuite: false
     unpack: true
-    debootstrap: Debian-Local Local-All Local-Arch
-    aptsources: Debian
+    debootstrap: Debian-Local Local-All Local-Arch ONL-Local Security-Local FRR-Local
+    aptsources: Debian ONL Security
 
   Debian:
     packages: *Packages
     source: http://${DEBIAN_MIRROR}
     suite: ${ONL_DEBIAN_SUITE}
-#    keyring: debian-archive-keyring
+    keyring: debian-archive-keyring
```

Comment:
Should this be `debian-keyring`? Genuine question, I don't know enough about debian, just noticed that both exist.